sssd_test_framework/roles/generic.py

```diff
@@ -300,20 +340,14 @@ def domain(self) -> str:
         """
         pass
 
+    @property
     @abstractmethod
-    def fqn(self, name: str) -> str:
+    def fqn(self) -> str:
```

Comment:
Don't you want the name to be passed in here to match how it's written elsewhere?

Reply:
Addressed in one of Pavel's comments.